docs/examples/kops-tests-private-net-bastion-host.md

```diff
@@ -0,0 +1,379 @@
+# USING KOPS WITH PRIVATE NETWORKING AND A BASTION HOST IN A HIGLY-AVAILABLE SETUP
+
+## WHAT WE WANT TO ACOMPLISH HERE ?.
+
+The exercise described on this document will focus on the following goals:
+
+- Demonstrate how to use a production-setup with 3 masters and two workers in different availability zones.
+- Demonstrate how to use a private networking setup with a bastion host.
+- Ensure our masters are deployed on 3 different AWS availability zones.
+- Ensure our nodes are deployed on 2 different AWS availability zones.
+- Add true high-availability to the bastion instance group.
+
+
+## PRE-FLIGHT CHECK:
+
```

Comment:
should we just have one doc with one preflight check?

Reply:
Done !. And modifying all the docs with a common requirements document